server/pageRenderer.js

```diff
@@ -120,6 +118,21 @@ export default function pageRenderer({
         ${styles}
       </head>
       <body>
+        <script>
+          ${
+            // If user has selected auto and device is in dark mode; ensure we update
+            // the theme before first render to screen
+            selectedTheme === 'auto'
+              ? `(function () {
+            try {
+              if (window.matchMedia('(prefers-color-scheme: dark)').matches === true) {
+                document.documentElement.setAttribute('data-theme', 'dark');
+              }
+            } catch (e) {}`
+              : ''
+          }
+          })();
+        </script>
```

Comment:
Might it be an idea to put the whole `script` tag in the ternary, so we don't end up with an empty tag that might confuse people?

Reply:
Skal være good nå